examples/aws/terraform/ha-autoscale-cluster/proxy_network.tf

```diff
@@ -202,6 +231,63 @@ resource "aws_lb_listener" "proxy_kube" {
   }
 }
 
+// MySQL port
```

Comment:
nit: I think just personal preference I would use [`for_each`](https://www.terraform.io/language/meta-arguments/for_each) instead of repeating 3 times for each database port.